pkg/simpleschema/transform_test.go

```diff
@@ -20,28 +20,15 @@ import (
 	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 )
 
-func TestBuildOpenAPISchema(t *testing.T) {
-	transformer := newTransformer()
-
-	// Load pre-defined types
-	err := transformer.loadPreDefinedTypes(map[string]interface{}{
-		"Address": map[string]interface{}{
-			"street":  "string",
-			"city":    "string",
-			"country": "string",
-		},
-		"Person": map[string]interface{}{
-			"name": "string",
-			"age":  "integer",
-		},
-	})
-	if err != nil {
-		t.Fatalf("Failed to load pre-defined types: %v", err)
-	}
+func floatPtr(f float64) *float64 {
```

Comment:
looks like this function isn't used, shall we remove it?

Reply:
Absolutely!